test/astro-prettier.test.ts

```diff
@@ -122,3 +54,101 @@ test.todo("properly follow prettier' advice on formatting comments");
+
+// https://prettier.io/docs/en/options.html#trailing-commas
+test('Can format an Astro file with prettier "trailingComma: none" option', Prettier, 'option-trailing-comma-none');
+
+// https://prettier.io/docs/en/options.html#bracket-spacing
+test('Can format an Astro file with prettier "bracketSpacing: true" option', Prettier, 'option-bracket-spacing-true');
+
+// https://prettier.io/docs/en/options.html#bracket-spacing
+test('Can format an Astro file with prettier "bracketSpacing: false" option', Prettier, 'option-bracket-spacing-false');
+
+// https://prettier.io/docs/en/options.html#bracket-line
+test('Can format an Astro file with prettier "bracketSameLine: false" option', Prettier, 'option-bracket-same-line-false');
+
+// https://prettier.io/docs/en/options.html#bracket-line
+test('Can format an Astro file with prettier "bracketSameLine: true" option', Prettier, 'option-bracket-same-line-true');
+
+// https://prettier.io/docs/en/options.html#arrow-function-parentheses
+test('Can format an Astro file with prettier "arrowParens: always" option', Prettier, 'option-arrow-parens-always');
+
+// https://prettier.io/docs/en/options.html#arrow-function-parentheses
+test('Can format an Astro file with prettier "arrowParens: avoid" option', Prettier, 'option-arrow-parens-avoid');
+
+// https://prettier.io/docs/en/options.html#prose-wrap
+test('Can format an Astro file with prettier "proseWrap: preserve" option', PrettierMarkdown, 'option-prose-wrap-preserve');
+
+// https://prettier.io/docs/en/options.html#prose-wrap
+test('Can format an Astro file with prettier "proseWrap: always" option', PrettierMarkdown, 'option-prose-wrap-always');
+
+// https://prettier.io/docs/en/options.html#prose-wrap
+test('Can format an Astro file with prettier "proseWrap: never" option', PrettierMarkdown, 'option-prose-wrap-never');
+
+// https://prettier.io/docs/en/options.html#html-whitespace-sensitivity
+test('Can format an Astro file with prettier "htmlWhitespaceSensitivity: css" option', Prettier, 'option-html-whitespace-sensitivity-css');
+
+// https://prettier.io/docs/en/options.html#html-whitespace-sensitivity
+test('Can format an Astro file with prettier "htmlWhitespaceSensitivity: strict" option', Prettier, 'option-html-whitespace-sensitivity-strict');
+
+// https://prettier.io/docs/en/options.html#html-whitespace-sensitivity
+test('Can format an Astro file with prettier "htmlWhitespaceSensitivity: ignore" option', Prettier, 'option-html-whitespace-sensitivity-ignore');
+
+// astro option: astroSortOrder
+test('Can format an Astro file with prettier "astroSortOrder: markup | styles" option', Prettier, 'option-astro-sort-order-markup-styles');
+
+// astro option: astroSortOrder
+test('Can format an Astro file with prettier "astroSortOrder: styles | markup" option', Prettier, 'option-astro-sort-order-styles-markup');
+
+// TODO: COMMENT OPTIONS https://prettier.io/blog/2018/11/07/1.15.0.html#whitespace-sensitive-formatting
```

Comment:
These could be all `test.todo()` calls, which is nice for tracking what still needs to be tested!

Reply:
I agree. But there are some options that I think should not be implemented like [file-path](https://prettier.io/docs/en/options.html#file-path) I let them there so everyone can see the available options, I was waiting for feedback. So let me know what you think about the options and I will change those lines to test.todo() calls.

Reply:
Ah, ok I interpreted those TODO comments as "here's some additional options that we don't have tests for, but that we should definitely implement tests for and then support for at some point"

Reply:
My bad, I use a vscode extension that highlights those comments for me. So about the missing options, what do you think?

Reply:
Mind making an RFC issue to track those options? I think this PR is fine to merge as-is for the moment, I don't think we need full options support in one PR